synapse_data_warehouse/synapse/dynamic_tables/V2.41.0__acl_latest_refactored.sql

```diff
@@ -0,0 +1,85 @@
+USE SCHEMA {{database_name}}.synapse; --noqa: JJ01,PRS,TMP
+
+
+CREATE OR REPLACE DYNAMIC TABLE ACL_LATEST
+    TARGET_LAG = '1 day'
+    WAREHOUSE = compute_xsmall
+    AS 
+    -- 1. Deduplicate the snapshots table based on each
+    -- entry's OWNER_ID, and select only the last 14 days'
+    -- worth of snapshots that have a non-empty RESOURCE_ACCESS column
+    WITH dedup_acl AS (
+        SELECT
+            CHANGE_TIMESTAMP,
+            CHANGE_TYPE,
+            CREATED_ON,
+            OWNER_ID,
+            OWNER_TYPE,
+            SNAPSHOT_DATE,
+            SNAPSHOT_TIMESTAMP,
+            RESOURCE_ACCESS
+        FROM {{ database_name }}.SYNAPSE_RAW.ACLSNAPSHOTS --noqa: TMP
+        WHERE
+            SNAPSHOT_DATE >= CURRENT_TIMESTAMP - INTERVAL '14 days'
+        AND
+            RESOURCE_ACCESS != '[]'
```

Comment:
If this isn't already documented in the ticket, can we get an inline comment explaining what it means for RESOURCE_ACCESS == '[]'

Reply:
Added! https://github.com/Sage-Bionetworks/snowflake/pull/140/commits/ee794901799db2d01ca67a24ae3ecf03e2eb42e3